docker-storage-setup.sh

```diff
@@ -632,7 +632,7 @@ canonicalize_block_devs() {
     # Looks like we allowed just device name (sda) as valid input. In
     # such cases /dev/$dev should be a valid block device.
     if [ -h "$dev" ];then
-      dev=$(readlink -f $dev)
+      dev=$(readlink -e $dev)
```

Comment:
What happens if redlink fails. If destination device does not exist. Will I get a proper error message about this.